quake1_example/class/_trigger.gd

```diff
@@ -1,50 +1,108 @@
 extends Area3D
 class_name QmapbspQuakeTrigger
+
 var props : Dictionary
 var v : QmapbspQuakeViewer
+var blocked : bool = false
+
+signal emit_message_state(m : String, show : bool)
+
+
 func _qmapbsp_is_brush_visible() -> bool : return false
 func _get_properties(dict : Dictionary) : props = dict
 
-signal emit_message_state(m : String, show : bool)
+
+func _flag(spawnflag : int) -> bool :
+	if not props.has(&'spawnflags') : 
+		printerr('Missing Spawnflags! ', spawnflag, ' ', self)
+		return false
+	var result : bool = _prop(&'spawnflags', 0) & spawnflag
+	return result
+
+
+func _prop(name : StringName, def) :
+	var result = props.get(name, def)
+	if typeof(result) == typeof(def) : return result
+	match typeof(def) :
+		TYPE_NIL:
+			result = def
+		TYPE_BOOL:
+			result = result.to_bool()
+		TYPE_INT:
+			result = result.to_int()
+		TYPE_FLOAT:
+			result = result.to_float()
+		TYPE_STRING:
+			result = str(result)
+		_ :
+			result = def
+	return result
+
 
 func _show_message_start(msg : String) :
 	emit_message_state.emit(msg, true)
 	
+	
 func _show_message_end() :
 	emit_message_state.emit('', false)
 
+
 func _ready() :
 	v = get_meta(&'viewer')
-	monitorable = false
+	set_deferred(&'monitorable', false)
```

Comment:
Saw this as suggestion by godot editor, as it caused errors sometimes when terminating app, did not see adverse effects .